tests/test_updatediff.py

```diff
@@ -366,7 +366,7 @@ def test_commit_hooks_respected(tmp_path_factory):
         git("commit", "-m", "commit 2")
         git("tag", "v2")
     # Update subproject to v2
-    copy(dst_path=dst1, defaults=True, overwrite=True)
+    copy(dst_path=dst1, defaults=True, overwrite=True, conflict="rej")
```

Comment:
Updated tests in this file to explicitly specify `rej` mode. This is not strictly needed, as `rej` is the default behavior, but this will prevent the tests from breaking if the default is later changed.